### PR TITLE
[SCFD-1041] Added temperature and pressure constraints for fluid properties

### PR DESCRIPTION
**Pull request by andrzej-krupka**

### CODE REVIEW

File: flow360/component/flow360_params/flow360_params.py [hunk 1]
```diff
@@ -706,9 +706,9 @@ class _FluidProperties(Flow360BaseModel):
     """
 
     temperature: TemperatureType = pd.Field()
-    pressure: PressureType = pd.Field()
-    density: DensityType = pd.Field()
-    viscosity: ViscosityType = pd.Field()
+    pressure: PressureType.NonNegative = pd.Field()
+    density: DensityType.NonNegative = pd.Field()
+    viscosity: ViscosityType.NonNegative = pd.Field()
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:711]: I guess 0 would not work. All should be positive

File: flow360/component/flow360_params/flow360_params.py [hunk 2]
```diff
@@ -730,7 +730,7 @@ class AirPressureTemperature(Flow360BaseModel):
     """
 
     model_type: Literal["AirPressure"] = pd.Field("AirPressure", alias="modelType", const=True)
-    pressure: PressureType = pd.Field()
+    pressure: PressureType.NonNegative = pd.Field()
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:733]: positive

File: flow360/component/flow360_params/flow360_params.py [hunk 3]
```diff
@@ -763,8 +763,8 @@ class AirDensityTemperature(Flow360BaseModel):
     """
 
     model_type: Literal["AirDensity"] = pd.Field("AirDensity", alias="modelType", const=True)
+    density: DensityType.NonNegative = pd.Field()
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:766]: positive

File: flow360/component/flow360_params/flow360_params.py (hunk 1 above)
Comment (andrzej-krupka) [reply flow360/component/flow360_params/flow360_params.py]: Correct, fixed to support conversion

File: flow360/component/flow360_params/flow360_params.py (hunk 2 above)
Comment (andrzej-krupka) [reply flow360/component/flow360_params/flow360_params.py]: Fixed

File: flow360/component/flow360_params/flow360_params.py (hunk 3 above)
Comment (andrzej-krupka) [reply flow360/component/flow360_params/flow360_params.py]: Fixed

**Review by maciej-flexcompute — APPROVED** [review]

**Pull request merged**